torch/csrc/jit/script/schema_type_parser.cpp

```diff
@@ -33,10 +33,14 @@ namespace script {
 TypeAndAlias SchemaTypeParser::parseBaseType() {
   static std::unordered_map<std::string, TypePtr> type_map = {
       {"Generator", GeneratorType::get()},
+      {"Dimname", StringType::get()},
+      {"DimnameList", ListType::create(StringType::get())}, // TODO Remove this type from the schema parser and write these as Dimname[] instead
```

Comment:
This one should not be here. It should be written as`Dimname[]`. We do not have aliases for any other types since it makes the entire type system seem more complicated than it is. We should change it to Dimname[] where they are used and teach the other pathways about it.

Reply:
fixed